runtime/src/dispatch.rs

```diff
@@ -0,0 +1,175 @@
+use castaway::cast;
+use std::marker::PhantomData;
+
+use fvm_ipld_encoding::ipld_block::IpldBlock;
+use fvm_ipld_encoding::RawBytes;
+use serde::{Deserialize, Serialize};
+
+use crate::ActorError;
+
+// TODO: Support non-cbor return values. This currently _always_ encodes return values to CBOR (or
```

Comment:
Please remove all the TODOs. Feel free to leave comments explaining limitations of the current implementation, and possibly point to an issue describing what to do about them. But TODOs are just frustrating to someone who runs into them.